test/unit_test/ccb_test.cc

```diff
@@ -18,92 +18,6 @@ void inject_slot_features(example* shared, example* slot);
 void remove_slot_features(example* shared, example* slot);
 }  // namespace CCB
 
-BOOST_AUTO_TEST_CASE(ccb_generate_interactions)
```

Comment:
are we removing this test?

Reply:
Yes, since the code this is testing is no longer needed